source/hands_on/cql.rst

```diff
@@ -0,0 +1,138 @@
+CQL
+^^^^^^^
+
+Overview
+---------
+Conservative Q-learning (CQL) was first proposed in `Conservative Q-Learning for Offline Reinforcement Learning <https://arxiv.org/abs/2006.04779>`_. Recent advances in reinforcement learning (RL), especially when combined with expressive deep network
```

Comment:
simplify this paragraph into two keypoints:
- motivation of offline RL
- definition of offline RL